Doc/library/datetime.rst

```diff
@@ -1591,26 +1596,25 @@ Instance methods:
 Example:
 
     >>> from datetime import time, tzinfo, timedelta
-    >>> class GMT1(tzinfo):
+    >>> class TZ1(tzinfo):
     ...     def utcoffset(self, dt):
     ...         return timedelta(hours=1)
     ...     def dst(self, dt):
     ...         return timedelta(0)
     ...     def tzname(self,dt):
-    ...         return "Europe/Prague"
+    ...         return "UTC+1"
```

Comment:
Super annoyingly, [POSIX-style offset names](https://github.com/dateutil/dateutil/issues/70#issuecomment-88982122) have an inverted sense, so `dateutil.parser.parse` (and I think `TZ` variables) would interpret `UTC+1` as the same as `-01:00`. Of course, many people use this in the opposite sense, which gives rise to some confusion.

I recommend bypassing the whole thing by returning "+01:00" here.